tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -108,3 +108,22 @@ Feature: full text search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: search files inside the current folder
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file with content "hello world inside root" to "file1.txt"
+    And user "Alice" has created folder "/Folder"
+    And user "Alice" has uploaded file with content "hello world inside Folder" to "/Folder/file1.txt"
+    And user "Alice" has created folder "/Folder/SubFolder"
+    And user "Alice" has uploaded file with content "hello world inside SubFolder" to "/Folder/SubFolder/file1.txt"
```

Comment:
```suggestion
    And user "Alice" has uploaded file with content "hello world inside SubFolder" to "/Folder/SubFolder/file2.txt"
```
I think different file name will be suitable

Reply:
i made the file name so that it is also listed because subfolder is inside the folder